src/misc.cpp

```diff
@@ -64,10 +62,18 @@ struct Tie: public streambuf { // MSVC requires split streambuf for cin and cout
 
 class Logger {
 
-  Logger() : in(cin.rdbuf(), file.rdbuf()), out(cout.rdbuf(), file.rdbuf()) {}
+  Logger()
+    :
+    in(std::cin.rdbuf(),
+       file.rdbuf()
+    ),
+    out(std::cout.rdbuf(),
+	file.rdbuf()
+    )
+    {}
```

Comment:
I don't think this kind of pedantic spacing improves readability. If you don't like compact one-liners, how about ?
```
Logger() :
    in(std::cin.rdbuf(), file.rdbuf()),
    out(std::cout.rdbuf(), file.rdbuf())
{}
```